Source/Utils/WarpXUtil.H

```diff
@@ -194,6 +194,38 @@ void getCellCoordinates (int i, int j, int k,
 #endif
 }
 
+/** \brief Compute physical coordinates (x,y,z) that correspond to a given (i,j,k) and
+ *  the corresponding staggering, mf_type when mf_type is of amrex::IntVect type.
+ *
+ * \param[in] i    index along x
+ * \param[in] j    index along y
+ * \param[in] k    index along z
+ * \param[in] mf_type IntVect containing the staggering type to convert (i,j,k) to (x,y,z)
+ * \param[in] domain_lo Physical coordinates of the lowest corner of the simulation domain
+ * \param[in] dx   Cell size of the simulation domain
+ *
+ * \param[out] x   physical coordinate along x
+ * \param[out] y   physical coordinate along y
+ * \param[out] z   physical coordinate along z
+ */
+AMREX_GPU_DEVICE AMREX_FORCE_INLINE
+void getCellCoordinatesIntVect (int i, int j, int k,
```

Comment:
what is the difference between this and the above get CellCoordinates?

Reply:
Discussed this during meeting. I have removed this function, and I am using getCellCoordinates function above throughout this PR.